atom/browser/native_window_mac.mm

```
@@ -1078,7 +1079,13 @@ static bool FromV8(v8::Isolate* isolate, v8::Handle<v8::Value> val,
       windowLevel = NSDockWindowLevel;
     }
   }
-  [window_ setLevel:windowLevel];
+
+  NSInteger newLevel = windowLevel + relativeLevel;
+  if (newLevel >= 0 && newLevel < CGWindowLevelForKey(kCGMaximumWindowLevelKey)) {
```

Yeah, I agree, the new level being out of range should throw an error.